CHANGELOG.md

```diff
@@ -0,0 +1,334 @@
+- Moves usage of default timezone from parser to evaluator [#448](https://github.com/partiql/partiql-lang-kotlin/pull/448)
+- [breaking-change] changes related to imported builders.
+
+### Fixed
+- Fixes struct handling of non-text struct field keys [#450](https://github.com/partiql/partiql-lang-kotlin/pull/450)
+
+## [0.2.7-alpha] - 2021-09-13
+
+### Fixed
+- Cherry picks "Fix bug causing multiple nested nots to parse very slowly [#436](https://github.com/partiql/partiql-lang-kotlin/pull/436) 
+  for `v0.2.7` release [#439](https://github.com/partiql/partiql-lang-kotlin/pull/439) 
+- Cherry picks "Use LazyThreadSafteyMode.PUBLICATION instead of NONE [#433](https://github.com/partiql/partiql-lang-kotlin/pull/433)
+  for `v0.2.7` release [#440](https://github.com/partiql/partiql-lang-kotlin/pull/440)
+
+## [0.1.7-alpha] - 2021-09-13
+
+### Fixed
+- Cherry picks "Fix bug causing multiple nested nots to parse very slowly [#436](https://github.com/partiql/partiql-lang-kotlin/pull/436) 
+  for `v0.1.7` release [#441](https://github.com/partiql/partiql-lang-kotlin/pull/441)
+- Cherry picks "Use LazyThreadSafteyMode.PUBLICATION instead of NONE [#433](https://github.com/partiql/partiql-lang-kotlin/pull/433)
+  for `v0.1.7` release [#442](https://github.com/partiql/partiql-lang-kotlin/pull/442)
+
+## [0.3.4-alpha] - 2021-09-10
+### Fixed
+- Bug causing multiple nested nots to parse very slowly [#436](https://github.com/partiql/partiql-lang-kotlin/pull/436)
+
+## [0.3.3-alpha] - 2021-09-09
+### Changed
+- Uses `LazyThreadSafteyMode.PUBLICATION` instead of NONE
+
+## [0.3.1-alpha] - 2021-06-18
+### Fixed
+- Prevent the `ORDER BY` clause from being dropped in visitor transforms [#420](https://github.com/partiql/partiql-lang-kotlin/pull/420)
+
+## [0.3.0-alpha] - 2021-06-09
+### Added
+- `DATE` and `TIME` data types
+- Enhancements made by/for DynamoDB
+- Compile-time `Thread.interrupted()` checks were added to help mitigate the impact of compiling extremely large SQL 
+  queries.
+- Various performance improvements to the compiler were added.
+
+### Changed
+- The modeling of `ExprNode` and `PartiqlAst` APIs has changed as needed to account for the enhancements to DML statements
+  and `ORDER BY`. Customers using these APIs may be affected.
+- Other minor API changes.
+
+### Fixed
+- Fixes parser for the top level tokens [#369](https://github.com/partiql/partiql-lang-kotlin/pull/369)
+- Make `SIZE` function work with s-expressions. [#379](https://github.com/partiql/partiql-lang-kotlin/pull/379)
+- A number of other minor bug fixes and technical debt has been addressed. For a complete list of PRs that made it into 
+  this release, please see the v0.3.0 GitHub milestone.
+
+## [0.1.6-alpha] - 2021-05-13
+
+### Fixed
+- Adds Compile-Time Thread.interrupted() checks [#398](https://github.com/partiql/partiql-lang-kotlin/pull/398)
+
+## [0.1.5-alpha] - 2021-04-27
```

Comment:
Similarly for these releases, no link appears.

- For `0.1.6-alpha`: https://github.com/partiql/partiql-lang-kotlin/compare/v0.1.5-alpha...v0.1.6-alpha
- For `0.1.5-alpha`: https://github.com/partiql/partiql-lang-kotlin/compare/v0.1.4-alpha...v0.1.5-alpha